README.md

```diff
@@ -25,7 +24,7 @@ Installing HDFS-Mesos on your Cluster
 --------------------------
 1. Upload `hdfs-mesos-*.tgz` to a node in your Mesos cluster (which is built to `$PROJ_DIR/build/hdfs-mesos-x.x.x.tgz`).
 2. Extract it with `tar zxvf hdfs-mesos-*.tgz`.
-3. Optional: Customize any additional configurations that weren't updated at compile time in `hdfs-mesos-*/etc/hadoop/*-site.xml` Note that if you update hdfs-site.xml, it will be used by the scheduler and bundled with the executors. However, core-site.xml and mesos-site.xml will be used by the scheduler only.
+3. Configure `hdfs-site.xml` and `mesos-site.xml`. They will eventually be transmitted to each executor and override any configuration bundled with the executor at compile time. However, `core-site.xml` may only be configured at compile time.
```

Comment:
Thanks for updating this.  This should still be marked as optional in case the user has already configured at compile time.